rocky/tests/test_organization.py

```diff
@@ -315,3 +315,22 @@ def test_admin_edits_organization(rf, admin_member, mocker):
     assertContains(resulted_response, "tags-color-1-light plain")  # default color
     assertContains(resulted_response, "tag1")
     assertContains(resulted_response, "tag2")
+
+
+def test_organization_code_validator(rf, superuser_member, mocker, mock_models_octopoes):
+    mocker.patch("katalogus.client.KATalogusClientV1")
+    if DENY_ORGANIZATION_CODES:
+        for organization_code in DENY_ORGANIZATION_CODES:
```

Comment:
The if is always true, so the if is useless here. I think it also enough to just test one denied code, testing all the code only makes the test suite longer for no gain.